artifacts/AdditionalSetupArtifacts.ps1

```diff
@@ -189,6 +149,34 @@ if ((Test-Path 'c:\run\cosmo.compiler.helper.psm1') -and ($env:IsBuildContainer)
     Import-Module 'c:\run\cosmo.compiler.helper.psm1' -DisableNameChecking -Force
 }
 
+Write-Host "Import BCContainerHelper"
+Write-Host "Install Nuget Provider"
+Install-PackageProvider -Name NuGet -MinimumVersion 2.8.5.201 -Force
```

Comment:
please check if there's at least 1 nuget artifact before installing all that stuff

Reply:
Nuget Tools are now Lazy Initialized